lib/beaker/host/unix/exec.rb

```diff
@@ -12,16 +12,17 @@ module Unix::Exec
   #
   # Will throw an exception RebootFailure if it fails
   def reboot(wait_time=10, max_connection_tries=9, uptime_retries=18)
+    require 'time'
+
     begin
-      original_uptime = exec(Beaker::Command.new("uptime"))
-      original_uptime_str = parse_uptime original_uptime.stdout
-      original_uptime_int = uptime_int original_uptime_str
+      original_boot_time_str = exec(Beaker::Command.new('who -b')).stdout
+      original_boot_time_line = original_boot_time_str.lines.grep(/system boot/).first
```

Comment:
Not sure if this is relevant, but the output on macOS is different:

```bash
# my local mac
╔ ☕️  gene:~
╚ᐅ who -b
reboot   ~        May  7 08:58 

# a test CentOS box
╔ ☕️  gene:~
╚ᐅ ssh gene-demobox.ops.puppetlabs.net who -b                                                       
         system boot  Dec 12 00:05
```

Reply:
Relaxed the regex check to just `boot`. Hopefully, that catches everything.